element_miniscope/imaging.py

```diff
@@ -167,6 +167,9 @@ def make(self, key):
             elif method == 'caiman':
                 loaded_caiman = loaded_result
                 key = {**key, 'processing_time': loaded_caiman.creation_time}
```

Comment:
consider removing the 2p methods.

Reply:
Thanks.  Removed Suite2p but kept CaImAn as it may be used for processing miniscope data.